### DISCUSSION TITLE
Moderation Policy

### DISCUSSION THREAD

**Discussion by AudacityTeam** [audacity/audacity#1354] [General]
We are initiating a moderation policy on this GitHub page, which we will be revising regularly. 

To keep it simple, our general approach is to avoid deleting comments, critical or otherwise unless we feel that it can be described as harassment or spam.

With that in mind, we may choose to delete comments:

1. When they introduce Issues not relevant to the discussion at hand. For example, an issue that cannot be addressed or resolved by anyone on the Audacity team
2. When they include very foul language intended to directly hurt someone
3. Spam (of various stripes that we are all familiar with)

---

Of course, it is possible to harass without breaking any of the above rules. For example, repeatedly bringing up the same issue and pasting it in multiple conversations on the same thread over and over again. We need to decide when to delete these comments on a case-by-case basis. Obviously, it is often necessary to have to repeat a point in an organic conversation but it can also be used as a tactic to achieve the goal of disrupting civil conversation. As a general principle: if we feel the purpose of a repeated comment is not in service of aiding or attempting to resolve a discussion, we may choose to delete it.

If users continuously falls foul of the above points, we may be forced to ban them. Generally, we do this only when a user behaves in a way we unanimously deem to be abusive.

**Comment by DoomRater** [comment]
When talking points don't involve the software misbehaving or features not yet implemented, should those be posted over here and not in the issues tracker?  I'm starting to see news feed posts coming from the issues tracker that, simply put, weren't issues with audacity.

**Reply by AudacityTeam** [reply]
It would certainly be desirable but we'd prefer to direct others as little as possible. We can always convert an issue to a discussion if we feel it is necessary.
